src/lib/grid-list/grid-list.spec.ts

```diff
@@ -294,12 +338,12 @@ class GridListWithUnspecifiedRowHeight { }
 
 @Component({template: `
     <div style="width:400px">
-      <md-grid-list cols="1" [rowHeight]="heightRatio">
+      <md-grid-list cols="1" [rowHeight]="rowHeight">
         <md-grid-tile></md-grid-tile>
       </md-grid-list>
     </div>`})
 class GirdListWithRowHeightRatio {
-  heightRatio: string;
+  rowHeight: string;
```

Comment:
Changed from **heightRatio** to **rowHeight** because it more clearly indicates direct binding to rowHeight.

I'm reusing the GirdListWithRowHeightRatio test component. Is there a rule that there should only be one test Component per test; should I create a new component for the test I added?